sample.ts

```diff
@@ -1,11 +1,17 @@
-import { Prism, Token } from "prismjs";
+import { Token as TokenType } from "prismjs/src/core.ts";
+
+let { Prism, Token } = await import("prismjs/src/core.ts");
```

Comment:
I don't know why, but if you remove `Token` from line 3, `tsx` stops compiling the code and throws with `SyntaxError: The requested module 'prismjs/src/core.ts' does not provide an export named 'Token'`.  🤷‍♂️🤦‍♂️